app/models/developer.rb

```diff
@@ -37,4 +39,9 @@ def self.queue
       .where(away: false)
       .order(max_updated_at: :asc)
   end
+
+  def status
+    return 'away' if away
+    'available'
```

Comment:
I think I'd prefer the syntax `away ? 'away' : 'available'` but I guess it's a matter of taste haha

Reply:
I'm not a ternary fan haha